### PR TITLE
Quick Sort C code improvement and resolved error #941

### LINKED ISSUE #941: Code Correction: Quick Sort
Quick Sort program written in c language requires some modification like swap function. With due permission, I can work on that.

### PR DESCRIPTION
**Pull request by geekyNehal**
Fixes #941 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Sample Input / Output.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

#### Changes proposed in this pull request:

- Error Resolved
- Code Improvement

#### Languages Used:
- C
- 

#### Files Added:
- Quick_Sort.c
-

### CODE REVIEW

**Comment by marwahmanbir** [comment]
Please complete the PR template above.

File: Quick_Sort/Quick_Sort.c [hunk 1]
```diff
@@ -30,32 +34,39 @@ void quickSort(int arr[], int low, int high)
...
 {
-    // freopen("in.txt","r",stdin);
-    // freopen("out.txt","w",stdout);
     int t;
+    printf("Enter Number of times you want to sort:\n");
     scanf("%d", &t);
 
     while(t--)
     {
         int n,i;
```
Comment (SakshiShreya) [inline Quick_Sort/Quick_Sort.c]: Add a space after all commas and around all operators.

**Comment by geekyNehal** [comment]
@SakshiShreya done changes stated above. Please tell me if more changes are to be done.

**Comment by geekyNehal** [comment]
@SakshiShreya Please tell me if any changes are required in the code.

**Comment by geekyNehal** [comment]
@jainaman224 changes done please review.

**Comment by jainaman224** [comment]
Please have a look at https://repl.it/@jainaman224/LightcyanDarkorangeCheckpoint.

**Comment by geekyNehal** [comment]
@jainaman224 sorry I was not getting this error on my terminal but resolved please review.

File: Quick_Sort/Quick_Sort.c [hunk 2]
```diff
@@ -1,27 +1,34 @@
...
+   *a = *b;
+   *b = temp;
+}
 int partition (int arr[], int low, int high)
 {
+    //Taking the last value of array as pivot.
     int pivot = arr[high];
     int i = (low - 1);
-    for (int j = low; j <= high- 1; j++)
+    for (int j = low; j <= high- 1 ; j++)
```
Comment (SakshiShreya) [inline Quick_Sort/Quick_Sort.c]: Fix the spacing - 
- Add space between `high` and `-`
- Remove space between `1` and `;`

File: Quick_Sort/Quick_Sort.c [hunk 3]
```diff
@@ -30,32 +37,41 @@ void quickSort(int arr[], int low, int high)
...
+        int n, i;
+        printf("Enter the Size of the Array:\n");
+        scanf("%d", &n);
+        int arr[n];
+        printf("Enter Array:\n");
+        for(i=0; i<n ; i++)
+            scanf("%d", &arr[i]);
+        quickSort(arr, 0, n-1) ;
+        printf("Sorted Array:\n");
+        for(i=0 ; i<n ; i++)
```
Comment (SakshiShreya) [inline Quick_Sort/Quick_Sort.c]: Add space around all operators except `++` in this line

**Comment by geekyNehal** [comment]
@SakshiShreya Please review

**Comment by geekyNehal** [comment]
@SakshiShreya please review

**Pull request merged**